payments-ui-core/detekt-baseline.xml

```diff
@@ -5,6 +5,7 @@
     <ID>ConstructorParameterNaming:CardNumberElement.kt$CardNumberElement$val _identifier: IdentifierSpec</ID>
     <ID>ConstructorParameterNaming:CvcElement.kt$CvcElement$val _identifier: IdentifierSpec</ID>
     <ID>CyclomaticComplexMethod:FormItemSpec.kt$FormItemSpecSerializer$override fun selectDeserializer(element: JsonElement): DeserializationStrategy&lt;FormItemSpec></ID>
+    <ID>CyclomaticComplexMethod:FormUI.kt$@Composable @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) fun FormUI( hiddenIdentifiers: Set&lt;IdentifierSpec>, enabled: Boolean, elements: List&lt;FormElement>, lastTextFieldIdentifier: IdentifierSpec?, modifier: Modifier = Modifier )</ID>
```

Comment:
Is this still needed?

Reply:
Function became too big so it was needed. I moved conditional `when` logic to a separate composable to fix it now.